eventuate-messaging-kafka-basic-consumer/src/main/java/io/eventuate/messaging/kafka/basic/consumer/EventuateKafkaConsumer.java

```diff
@@ -163,6 +174,7 @@ private void runPollingLoop(KafkaConsumer<String, String> consumer, KafkaMessage
 
   public void stop() {
     stopFlag.set(true);
+//    consumer.close(Duration.of(200, ChronoUnit.MILLIS)); //can produce java.util.ConcurrentModificationException: KafkaConsumer is not safe for multi-threaded access
```

Comment:
What's this?

Reply:
Just a reminder why we are not closing consumer here.

Reply:
The line is commented out. Perhaps 

```
// Can't call consumer.close() here because its not threadsafe.

Reply:
ok, I will change to that